src/ImageSharp/Formats/WebP/LosslessUtils.cs

```diff
@@ -0,0 +1,656 @@
+                output[x] = AddPixels(input[x], pred);
+            }
+        }
+
+        private static void PredictorAdd8(Span<uint> input, int startIdx, int numberOfPixels, int width, Span<uint> output)
+        {
+            int endIdx = startIdx + numberOfPixels;
+            int offset = 0;
+            for (int x = startIdx; x < endIdx; ++x)
+            {
+                uint pred = Predictor8(output[x - 1], output, startIdx - width + offset++);
+                output[x] = AddPixels(input[x], pred);
+            }
+        }
+
+        private static void PredictorAdd9(Span<uint> input, int startIdx, int numberOfPixels, int width, Span<uint> output)
+        {
+            int endIdx = startIdx + numberOfPixels;
+            int offset = 0;
+            for (int x = startIdx; x < endIdx; ++x)
+            {
+                uint pred = Predictor9(output[x - 1], output, startIdx - width + offset++);
+                output[x] = AddPixels(input[x], pred);
+            }
+        }
+
+        private static void PredictorAdd10(Span<uint> input, int startIdx, int numberOfPixels, int width, Span<uint> output)
+        {
+            int endIdx = startIdx + numberOfPixels;
+            int offset = 0;
+            for (int x = startIdx; x < endIdx; ++x)
+            {
+                uint pred = Predictor10(output[x - 1], output, startIdx - width + offset++);
+                output[x] = AddPixels(input[x], pred);
+            }
+        }
+
+        private static void PredictorAdd11(Span<uint> input, int startIdx, int numberOfPixels, int width, Span<uint> output)
+        {
+            int endIdx = startIdx + numberOfPixels;
+            int offset = 0;
+            for (int x = startIdx; x < endIdx; ++x)
+            {
+                uint pred = Predictor11(output[x - 1], output, startIdx - width + offset++);
+                output[x] = AddPixels(input[x], pred);
+            }
+        }
+
+        private static void PredictorAdd12(Span<uint> input, int startIdx, int numberOfPixels, int width, Span<uint> output)
+        {
+            int endIdx = startIdx + numberOfPixels;
+            int offset = 0;
+            for (int x = startIdx; x < endIdx; ++x)
+            {
+                uint pred = Predictor12(output[x - 1], output, startIdx - width + offset++);
+                output[x] = AddPixels(input[x], pred);
+            }
+        }
+
+        private static void PredictorAdd13(Span<uint> input, int startIdx, int numberOfPixels, int width, Span<uint> output)
+        {
+            int endIdx = startIdx + numberOfPixels;
+            int offset = 0;
+            for (int x = startIdx; x < endIdx; ++x)
+            {
+                uint pred = Predictor13(output[x - 1], output, startIdx - width + offset++);
+                output[x] = AddPixels(input[x], pred);
+            }
+        }
+
+        private static uint Predictor0()
```

Comment:
why not using a const here as well (private const uint Predictor0 = WebPConstants.ArgbBlack?